test/test_nn.py

```diff
@@ -11573,6 +11573,17 @@ def test_cross_entropy_large_tensor(self, device, reduction):
         if reduction != "none":
             self.assertTrue(torch.allclose(logits.grad.cpu(), logits_cpu.grad, rtol=rtol, atol=atol))
 
+    @parametrize_test("reduction", ("none", "mean", "sum"))
+    def test_cross_entropy_dim_parameter(self, device, reduction):
+        loss = nn.CrossEntropyLoss(reduction=reduction)
+        input_1 = torch.randn(15, 10, 20, dtype=torch.float, device=device)
+        input_2 = input_1.swapaxes(2, 1)
+        target = torch.empty(15, 20, dtype=torch.long, device=device).random_(10)
+
+        loss_1 = loss(input_1, target)
+        loss_2 = loss(input_2, target, dim=2)
```

Comment:
`dim` should be passed during construction of the `nn.CrossEntropyLoss()` object instead of during the forward pass, no?

Reply:
@jbschlosser, Apologies, my bad, I've corrected it now.